app/scripts/lib/app-start.js

```diff
@@ -506,28 +509,40 @@ function (
 
     /**
      * Sets a UUID value that is unrelated to any account information.
-     * This value is useful to determine if the logged out user qualifies for ab testing or metrics.
+     * This value is useful to determine if the logged out user qualifies
+     * for A/B testing or metrics.
      */
     initializeUniqueUserId: function () {
-      var storage = Storage.factory('localStorage', this._window);
-
       var uniqueUserId;
 
-      // Reuse a stored unique user id if available.
-      if (storage.get('uuid')) {
-        // stomlinson on 2015-07-08:
-        // `uuid` is the old name, this is transition code
-        // and can hopefully be removed after a time.
-        uniqueUserId = storage.get('uuid');
-        storage.remove('uuid');
-      } else if (storage.get('uniqueUserId')) {
-        // uniqueUserId is the new name.
-        uniqueUserId = storage.get('uniqueUserId');
-      } else {
-        uniqueUserId = uuid.v4();
+      if (this._searchParam('resume')) {
```

Comment:
This is a lot of initialization code in app-start.js, could the UniqueUserId live as its own model that is tracked by the User model?